datashare-app/src/main/java/org/icij/datashare/web/BatchSearchResource.java

```diff
@@ -37,13 +39,15 @@ public class BatchSearchResource {
     private final BatchSearchRepository batchSearchRepository;
     private final BlockingQueue<String> batchSearchQueue;
     private final PropertiesProvider propertiesProvider;
+    private final PayloadFormatter payloadFormatter;
     private final int MAX_BATCH_SIZE = 60000;
 
     @Inject
     public BatchSearchResource(final BatchSearchRepository batchSearchRepository, BlockingQueue<String> batchSearchQueue, PropertiesProvider propertiesProvider) {
         this.batchSearchRepository = batchSearchRepository;
         this.batchSearchQueue = batchSearchQueue;
         this.propertiesProvider = propertiesProvider;
+        this.payloadFormatter = new PayloadFormatter();
```

Comment:
This one can be removed

Reply:
Good catch! I need to rebase this branch to use the static method instead.